### PR TITLE
[data] teh -> the

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: data/pathfinder_pfs/paizo/pathfinder_society/pfs_chronicle_sheets/scenarios/season_6/pfscs_S6_equip.lst [hunk 1]
```diff
@@ -25,7 +25,7 @@ Blade of the open road (lesser).MOD																TYPE:PFS						COST:19715	WT:4
...
...																																												DESC:Blades of the open road are longswords specifically crafted to strike against the Aspis Consortium. A lesser blade of the open road is a +1 longsword with a Glyph of the Open Road stamped into its hilt. A blade of the open road is a +1 Aspis bane longsword. Against agents of the Aspis Consortium and creatures directly employed by the Aspis Consortium, the weapon's enhancement bonus is 2 higher than its actual bonus, and the weapon deals an extra 2d6 points of damage. A greater blade of the open road has all the properties of the blade of the open road and gains the ominous and heartseeker weapon special abilities against the Aspis Consortium agents and the COnsortium's employees. Any wayfinder the carrier the wielder carries grants a +2 circumstance bonus on all Survival checks, not just checks to avoid becoming lost. A blade of the open road and greater blade of the open road increase this bonus by +4 or +6, respectively. Between scenarios, the wielder can enhance a lesser blade of the open road to become a blade of the open road and later into a greater blade of the open road by paying the difference in price between the two items.
```
Comment (LegacyKing) [inline data/pathfinder_pfs/paizo/pathfinder_society/pfs_chronicle_sheets/scenarios/season_6/pfscs_S6_equip.lst]: I'm tired, I don't see the change on this line.

Comment (grimreaper) [inline data/pathfinder_pfs/paizo/pathfinder_society/pfs_chronicle_sheets/scenarios/season_6/pfscs_S6_equip.lst]: "employed by teh Aspis Consortium"

Comment (grimreaper) [inline data/pathfinder_pfs/paizo/pathfinder_society/pfs_chronicle_sheets/scenarios/season_6/pfscs_S6_equip.lst]: "employed by teh Aspis Consortium"

Comment (Zaister) [inline data/pathfinder_pfs/paizo/pathfinder_society/pfs_chronicle_sheets/scenarios/season_6/pfscs_S6_equip.lst]: The second file also has "a force for goon" in the changed line :)

Comment (grimreaper) [inline data/pathfinder_pfs/paizo/pathfinder_society/pfs_chronicle_sheets/scenarios/season_6/pfscs_S6_equip.lst]: changed

**Pull request merged**